packages/profile-sync-controller/package.json

```
@@ -136,8 +136,8 @@
     "@metamask/accounts-controller": "^27.0.0",
     "@metamask/keyring-controller": "^21.0.0",
     "@metamask/network-controller": "^23.0.0",
-    "@metamask/providers": "^18.1.0",
-    "@metamask/snaps-controllers": "^9.19.0",
+    "@metamask/providers": "^21.0.0",
+    "@metamask/snaps-controllers": "^11.2.1",
```

Would it be possible to require just the major version instead (that's what we usually do) ? The same update should be made for the other peer dependency bumps for this package, along with the corresponding changelog updates.
```suggestion
    "@metamask/snaps-controllers": "^11.0.0",